testing/run_tests.py

```diff
@@ -336,10 +336,13 @@ def RunJavaTests(filter, android_variant='android_debug_unopt'):
   gradle_cache_dir = os.path.join(out_dir, android_variant, 'robolectric_tests', '.gradle')
 
   test_class = filter if filter else 'io.flutter.FlutterTestSuite'
+  ignore_deprecations = 'true' if ignore_android_deprecations else ''
```

Comment:
Might be a bit more clear if `default=True` were set in the parser.add_argument rather than depend on the value being None.